lib/core/src/send_swap.rs

```diff
@@ -74,8 +74,8 @@ impl SendSwapStateHandler {
             Ok(SubSwapStates::InvoiceSet) => {
                 match (swap.state, swap.lockup_tx_id.clone()) {
                     (PaymentState::Created, None) | (PaymentState::TimedOut, None) => {
-                        let create_response = swap.get_boltz_create_response()?;
-                        let lockup_tx = self.lockup_funds(id, &create_response).await?;
+                        let create_response = swap.get_boltz_create_response().unwrap();
+                        let lockup_tx = self.lockup_funds(id, &create_response).await.unwrap();
```

Comment:
Same with the `unwrap()`, is there a reason?

Reply:
Good catch Ross, I used those unwraps for debugging, thanks